src/main/java/mk/learning/fileshare/Services/ReadExcel.java

```diff
@@ -0,0 +1,47 @@
+package mk.learning.fileshare.Services;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.DataFormatter;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ReadExcel {
+
+	public static Map<String, Map<String, String>> setMapData(String Filename) throws IOException {
```

Comment:
Remove static.
Return value must be boolean. True on successfully parsing excel and loading data on hashmap. false on error in processing